.github/workflows/forge-fmt.yml

```diff
@@ -0,0 +1,26 @@
+name: Forge format
+
+on:
+  workflow_call:
+    secrets:
+      MACHINE_USER_PRIVATE_KEY:
+        required: true
```

Comment:
You successfully added the workflow but you never call it ^^
You either need to trigger it on PRs, or call it explicitly from another workflow